pkgs/development/python-modules/pylibgen/default.nix

```diff
@@ -1,31 +1,33 @@
-{ buildPythonPackage, lib, fetchPypi
+{ lib, buildPythonPackage, fetchFromGitHub
 , isPy3k
 , requests
 , pytest
+, pre-commit
 }:
 
 buildPythonPackage rec {
   pname = "pylibgen";
-  version = "2.0.1";
+  version = "2.0.2";
+  disabled = !isPy3k;
```

Comment:
This is a little more strict https://github.com/joshuarli/pylibgen/blob/master/setup.py#L44

Reply:
hmm that's my bad, i stopped testing for 3.5, and 3.6 due to all the package builds :(